internal/bminventory/inventory.go

```diff
@@ -114,10 +116,41 @@ const ignitionConfigFormat = `{
       "path": "/etc/motd",
       "mode": 644,
       "contents": { "source": "data:,{{.AGENT_MOTD}}" }
-    }]
+	}]
   }
 }`
 
+const onPremIgnitionConfigFormat = `{
```

Comment:
Would be nice if we didn't have to duplicate ignitionConfigFormat.  It's a bigger change but we should try to avoid duplication between the different environments.  One way would be to define the yaml as a struct and simply add another file to the list if needed, then marshall it.

Reply:
Yup, that's the plan I am working on. This was a temp change added. Thank you.

Reply:
@filanov Done. Please take a look 4804bdc